modules/oadp-installing-dpa.adoc

```diff
@@ -212,28 +242,19 @@ spec:
         default: true
         credential:
           key: cloud
-          name: {credentials}
+          name: <default_secret> <.>
         objectStorage:
           bucket: <bucket_name> <.>
           prefix: <prefix> <.>
 ----
-<.> Specify the default plug-in for the backup provider, for example, `gcp`, if it exists.
+<.> Specify the default plug-in for the backup provider, for example, `gcp`, if appropriate.
 <.> Specify the `csi` default plug-in if you use CSI snapshots to back up PVs. The volume locations are configured in the `VolumeSnapshotClass` manifest.
 <.> Mandatory for CSI.
-endif::[]
-<.> Set to `false` to disable the Restic installation. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
-ifdef::installing-oadp-ocs[]
+<.> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that an environment with many nodes will have a lot of `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
```

Comment:
same as above. 
1. `daemon set,` --> `DaemonSet`

2. 
`which means that an environment with many nodes will have a lot of Restic pods running.`
-->
`which means that an environment with each worker nodes will have a Restic pods running.`